src/Host.cpp

```diff
@@ -764,33 +767,57 @@ bool Host::installPackage(const QString& fileName, int module )
         mpEditorDialog->doCleanReset();
     }
     QFile file2;
-    if( fileName.endsWith(".zip") || fileName.endsWith(".mpackage") )
+    if(  fileName.endsWith( QStringLiteral( ".zip" ), Qt::CaseInsensitive )
+      || fileName.endsWith( QStringLiteral( ".mpackage"), Qt::CaseInsensitive ) )
     {
-        QString _home = QDir::homePath();
-        _home.append( "/.config/mudlet/profiles/" );
-        _home.append( getName() );
-        QString _dest = QString( "%1/%2/").arg( _home ).arg( packageName );
-        QDir _tmpDir;
-        _tmpDir.mkpath(_dest);
-
-        QUiLoader loader;
-        QFile file(":/ui/package_manager_unpack.ui");
-        file.open(QFile::ReadOnly);
-        pUnzipDialog = dynamic_cast<QDialog *>(loader.load( &file, 0 ) );
-        file.close();
-        if( ! pUnzipDialog ) return false;
-
-        pUnzipDialog->setWindowTitle( tr( "Unpacking package: %1" ).arg( fileName ) );
+        QString _home = QStringLiteral( "%1/.config/mudlet/profiles/%2" )
+                        .arg( QDir::homePath() )
+                        .arg( getName() );
+        QString _dest = QStringLiteral( "%1/%2/" )
+                        .arg( _home )
+                        .arg( packageName );
+        QDir _tmpDir( _home ); // home directory for the PROFILE
+        _tmpDir.mkpath( _dest );
```

Comment:
Previously setting `_tmpDir` to the profile directory ensures that any _relative_ directory is resolved relative to the profile's home directory. Previously it would have been relative to the Application's execution directory which is OS dependent IIRC...